python/lsst/ap/association/transformDiaSourceCatalog.py

```diff
@@ -187,7 +189,7 @@ def run(self,
             Filter band of the science image.
         ccdVisitId : `int`
             Identifier for this detector+visit.
-        funcs : `lsst.pipe.tasks.functors.Functors`
+        funcs : `lsst.pipe.tasks.functors.Functors`, optional
             Functors to apply to the catalog's columns.
```

Comment:
Missing the new `idGenerator` docstring.

Reply:
I think I must have ultimately intended to remove the `idGenerator` argument instead, since it isn't actually used in `run` and I didn't mark `ccdVisitId` as deprecated.  I think that's because it uses just the `ccdVisitId` (instead of e.g. also making an `lsst.afw.table.IdFactory`) and I was going for minimal changes.  If you'd prefer for me to deprecate `ccdVisitId` and pass an `idGenerator` instance for better consistency with other tasks, I can do that instead.

Reply:
Good question! I think since we don't need the packer itself and this operates per-detector/visit, we can keep it as-is. Also, all that's used for is a log message and setting the value in the table.